docs/CLERK_SETUP.md

```
@@ -0,0 +1,200 @@
+**IMPORTANT**: `.env.local` is in `.gitignore` and should **NEVER** be committed to version control!
+
+### Step 2: Fill in Your Clerk Keys
+
+Open `.env.local` and replace the placeholders with your actual Clerk keys:
+
+```env
+# Clerk Authentication
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_YOUR_ACTUAL_KEY_HERE
+CLERK_SECRET_KEY=sk_test_YOUR_ACTUAL_SECRET_HERE
+```
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)